btb_phylo.py

```diff
@@ -716,10 +716,8 @@ def parse_args():
     subparser.add_argument("--all_wgs_samples_filepath", help="path to \
         'all_wgs_samples' .csv file", \
             default=utils.DEFAULT_WGS_SAMPLES_FILEPATH)
-    subparser.add_argument("--pcmapped", "-pc", dest="pcMapped", \
-        type=float, nargs=2, help="optional filter", default=(90, 100))
     subparser.add_argument("--flag", "-f", dest="flag", nargs="+", \
-        help="optional filter", default=["BritishbTB"])
+        help="optional filter", default=["nonBritishbTB", "BritishbTB"])
```

Comment:
Not sure you need this filter at all.  This flag is generated by Javi's python script which assigns clades.  It add the 'nonBritishbTB' flag is the matching shows some discrepancies.

Reply:
Have removed this filter